src/tocsym.c

```diff
@@ -285,7 +290,10 @@ Symbol *toSymbol(Dsymbol *s)
                 Symbol *s = symbol_calloc(id);
                 slist_add(s);
 
-                s->prettyIdent = fd->toPrettyChars(true);
+                if (global.params.isWindows)
+                    s->prettyIdent = fd->toPrettyChars(true);
+                else
+                    s->prettyIdent = fd->toChars();
                 s->Sclass = SCglobal;
```

Comment:
Ah, I confused this with FunctionType::toChars, FuncDeclaration::toChars uses Dsymbol::toChars and should be fine.